### PR TITLE
[FIX] mail: fix to add reaction on a message

### PR DESCRIPTION
**Pull request by jgi-odoo**
Small fix when adding a reaction to a message

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mail/static/src/models/emoji_list_view/emoji_list_view.js [hunk 1]
```diff
@@ -16,10 +16,10 @@ registerModel({
          * @param {MouseEvent} ev 
          */
         onClickEmoji(ev) {
-            if (this.popoverViewOwner.messageActionListOwnerAsReaction) {
+            if (this.popoverViewOwner && this.popoverViewOwner.messageActionListOwnerAsReaction) {
                 this.popoverViewOwner.messageActionListOwnerAsReaction.onClickReaction(ev);
             }
-            if (this.popoverViewOwner.composerViewOwnerAsEmoji) {
+            if (this.popoverViewOwner && this.popoverViewOwner.composerViewOwnerAsEmoji) {
```
Comment (alexkuhn) [inline addons/mail/static/src/models/emoji_list_view/emoji_list_view.js:22]: Prefer `} else if {` for the second conditional branching. You should no longer need guard

File: addons/mail/static/src/models/emoji_list_view/emoji_list_view.js [hunk 2]
```diff
@@ -13,13 +13,12 @@ registerModel({
     },
     recordMethods: {
         /**
-         * @param {MouseEvent} ev 
-         */
+         * @param {MouseEvent} ev
+        */
         onClickEmoji(ev) {
-            if (this.popoverViewOwner.messageActionListOwnerAsReaction) {
+            if (this.popoverViewOwner && this.popoverViewOwner.messageActionListOwnerAsReaction) {
```
Comment (alexkuhn) [inline addons/mail/static/src/models/emoji_list_view/emoji_list_view.js:19]: no `this.popoverViewOwner &&`

Comment (seb-odoo) [inline addons/mail/static/src/models/emoji_list_view/emoji_list_view.js:19]: change not needed

File: addons/mail/static/src/models/emoji_list_view/emoji_list_view.js [hunk 3]
```diff
@@ -13,13 +13,12 @@ registerModel({
     },
     recordMethods: {
         /**
-         * @param {MouseEvent} ev 
-         */
+         * @param {MouseEvent} ev
+        */
```
Comment (seb-odoo) [inline addons/mail/static/src/models/emoji_list_view/emoji_list_view.js:17]: wrong indent

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**